nb/o.n.upgrader/src/org/netbeans/upgrade/AutoUpgrade.java

```diff
@@ -65,10 +63,6 @@ public static void main (String[] args) throws Exception {
         
         // try new place
         File sourceFolder = checkPreviousOnOsSpecificPlace (NEWER_VERSION_TO_CHECK);
-        if (sourceFolder == null) {
-            // try former place
-            sourceFolder = checkPrevious (VERSION_TO_CHECK);
```

Comment:
i think `checkPrevious` is now no longer used and can be removed, same with `upgradeBuildProperties`, `doNonStandardUpgrade`, potentially more

Reply:
True!  Done heavier cleanup, and resolved a couple of other issues that were hinted - try-with-resources (missing close on exception) and new Integer.